code/src/Components/TaskItem.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from "react"
+import { useDispatch } from "react-redux"
+
+import { todo } from "../Reducers/todo"
+
+export const TaskItem = ({ item }) => {
+  const dispatch = useDispatch();
+  const [checked, setChecked] = useState(item.isComplete)
+  const handleCheckboxClick = () => {
+    setChecked();
+    dispatch(todo.actions.setComplete(item.id))
+  }
+
+  //Deleting one element from the redux store
+  const onItemDelete = (id) => {
+    dispatch(todo.actions.removeItem({ id: item.id }))
```

Comment:
```suggestion
    dispatch(todo.actions.removeItem({ item.id }))
```

as you did on line 11.

Not 100% if I think right here since you passing in an argument in this case. I'm curious if it will work with my suggestion.

Reply:
It gets angry when i try this :(

Reply:
You can either have it as you did or update the reducer so that it understands that the payload = id.